controller_manager/src/controller_manager.cpp

```diff
@@ -24,7 +24,7 @@
 
 #include "lifecycle_msgs/msg/state.hpp"
 
-#include "rcutils/logging_macros.h"
```

Comment:
With `rcutils` not being used here, we can also lift the dependency to `rcutils` in the `package.xml`, can't we?